.travis/test.sh

```diff
@@ -14,4 +14,13 @@ elif [[ "$TEST_TYPE" == "bench" ]]; then
     export MEMENTO_CURR_COMMIT="HEAD"
     export MEMENTO_BASE_COMMIT="origin/master"
     julia -e 'Pkg.test("Memento"; coverage=true)'
+elif [[ "$TEST_TYPE" == "userimage" ]]; then
```

Comment:
Might be worth adding a few comments as this is a bit of a weird test case that most packages won't be doing.